gems/actionview/CVE-2016-0752.yml

```diff
@@ -84,12 +84,9 @@ description: |
   Credits
   -------
   Thanks John Poulin for reporting this!
-
 cvss_v2: 5.0
 cvss_v3: 7.5
-
-# "~> 3.2.22.1" is found in gems/actionpack/CVE-2016-0752.yml
```

Comment:
It looks like `load/dump` removes comments. Should these be converted to a `comments` field or something similar so they are kept around?